Model/CcConfigProvider.php

```diff
@@ -106,7 +109,8 @@ public function getConfig()
                 'reach_cc' => [
                     'oc_enabled'=>(boolean)$this->reachHelper->getAllowOpenContract(),
                     'open_contracts'=>$this->getOpenContracts(),
-                     //got idea from here: https://webkul.com/blog/adding-additional-variables-in-window-checkoutconfig-on-magento-2-checkout-page/
+                    'selected_currency'=>$this->_storeManager->getStore()->getCurrentCurrency()->getCode(),
```

Comment:
Oh nice! you were able to determine where the currency is being stored!

Reply:
Well, sorta. At the time the contracts are being rendered on the FE the selected currency is not available. But this line of code solves that